README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://ci.centos.org/buildStatus/icon?job=gluster_samba-integration)](https://ci.centos.org/view/Gluster/job/gluster_samba-integration/)
```

Comment:
This shows the status of the last build, which includes builds for PRs. I do not know how to pass the branch=master to the URL to get the image. Maybe something to investigate?

Reply:
ah... good point!